mir_connector/inorbit_mir_connector/src/connector.py

```diff
@@ -187,14 +203,20 @@ def _inorbit_command_handler(self, command_name, args, options):
     def _connect(self) -> None:
         """Connect to the robot services and to InOrbit"""
         super()._connect()
+        # If enabled, initiate the websockets client
         if self.ws_enabled:
             self.mir_ws.connect()
+        # Start garbage collection for missions
+        self._missions_gc_process = multiprocessing.Process(target=self._missions_garbage_collector)
```

Comment:
Hey, since the connector isn't CPU bounded, do you think the multi process approach has any advantage over threads? My concern is that a new process will require probably more RAM and inherit some resources from the original process that may introduce unexpected concurrency issues in the future :thinking: